pom.xml

```diff
@@ -2726,6 +2726,26 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
```

Comment:
Can we have a documentation somewhere of how to use this plugin?

Reply:
It is quite straightforward: if it finds an error, the compilation breaks

Reply:
Ok thanks, it is runned by default